### PR TITLE
added an example data set with name and URL

### PR DESCRIPTION
**Pull request by ebarry**

### CODE REVIEW

File: add-dataset/README.md [hunk 1]
```diff
@@ -25,21 +25,42 @@ After going through this tutorial you will know how to:
...
+* Copy the URL of the dataset you would like to archive
+* Visit the Data Together [Add a Dataset](#) page
+* Paste the URL into the "URL to archive" text box
+* Type a simple description in your own words into the "How do you use this dataset?" text box 
+* Click "Archive" button
+* Success! You archived a dataset :)
+
+<img src="images/screenshot-AddADataset.png" width=500>
+
+#### Step 4. Review the actions you can take on datasets
```
Comment (dcwalk) [inline add-dataset/README.md:53]: Do we want this here? Or pulled out from the tutorial

**Review by dcwalk — COMMENTED** [review]
This is a great next iteration! 
I'm happy to redeploy with this if that works for you :)

**Comment by dcwalk** [comment]
Thanks @ebarry! ❤️ I would love to deploy with this then do another round, maybe pulling the ellipses section to the next iteration?

File: add-dataset/README.md [hunk 2]
```diff
@@ -25,21 +25,42 @@ After going through this tutorial you will know how to:
...
 
-#### Step 2. ??
+<img src="images/Screenshot-signup.png" width=250> <img src="images/screenshot-signup-1.png" width=250>
 
-### Register on Data Together (Optional){#lesson-two}
 
-1.
+#### Step 3. Archive a dataset on Data Together
+
+* Copy the URL of the dataset you would like to archive
```
Comment (mhucka) [inline add-dataset/README.md:44]: Maybe clarify that "copy the URL" means for people to copy it to their clipboard?

File: add-dataset/README.md [hunk 3]
```diff
@@ -25,21 +25,42 @@ After going through this tutorial you will know how to:
...
+* Visit the Data Together [Add a Dataset](#) page
+* Paste the URL into the "URL to archive" text box
+* Type a simple description in your own words into the "How do you use this dataset?" text box 
+* Click "Archive" button
```
Comment (mhucka) [inline add-dataset/README.md]: Add "the" after "click".

**Review by mhucka — COMMENTED** [review]
Very minor suggestions.

**Review by dcwalk — APPROVED** [review]

**Pull request merged**